app/views/directives/osc-routing-service.html

```diff
@@ -0,0 +1,67 @@
+<ng-form name="routingServiceForm">
+  <div class="row">
+    <div class="form-group" ng-class="{ 'col-sm-6': showWeight, 'col-sm-12': !showWeight }">
+      <label for="{{id}}-service-select" class="required">
+        Service
+      </label>
+      <select
+          id="{{id}}-service-select"
+          ng-model="model.service"
+          ng-options="service as service.metadata.name for service in services"
+          required
+          class="form-control"
+          ng-attr-aria-describedby="{{id}}-service-help">
+      </select>
+      <div>
+        <span ng-attr-id="{{id}}-service-help" class="help-block">
+          <span ng-if="!isAlternate">Service to route to.</span>
+          <span ng-if="isAlternate">Alternate service for route traffic.</span>
+        </span>
+      </div>
+      <div ng-if="(services | hashSize) === 0" class="has-error">
+        <span class="help-block">
+          There are no <span ng-if="is-alternate">additional</span> services in your project to expose
```

Comment:
what if there is only one service in the project?  you can route to that single service in the to, but would have no more to pick from to be "additional",  services|hashSize would be zero and this help block wouldnt show, unless im missing something